extensions/summary/summary.md

```diff
@@ -0,0 +1,85 @@
+# Collection Summary
+
+This proposal suggests that the spec define a `summary` resource (all names TBD) that describes a collection.  It may be that this information belongs in the existing collection info (e.g. at `/collections/{id}`), but for the purposes of this doc, we'll say it is going to be a new resource (e.g. at `/collections/{id}/summary`).
+
+There are multiple motivations for this:
+
+ * A client should have a way to get the formal schema for items in a collection if one exists.  The current suggestion to use a link with `rel="describedBy"` doesn't fully address this need since it doesn't provide a robust way for a client to get the schema for multiple content types (for example, if a service supports GML, KML, and GeoJSON output, the `type` and `rel` of a link is not enough information for a client to know which schema applies to which output format).
+
+ * A "filter" extension should have a way to describe which properties are filterable and how clients should access a specific property in a filter expression.
+
+ * A "search" extension should have a way to describe which properties can be used in sorting.
+
+ * An "editing" extension should have a way to describe which properties are writeable (e.g. in an update) or might want to allow for a different schema for each method (e.g. one schema for create, one for update).
+
+## Example
+
+Here is an example of what this summary resource might look like for a "buildings" collection (e.g. at https://example.com/collections/buildings/summary)
+
+```json
+{
+  "properties": {
+    "application/geo+json": {
+      "height": {
+        "schema": {
+          "href": "http://example.com/collections/buildings/schema#/definitions/height"
+        }
+      },
+      "age": {
+        "schema": {
+          "href": "http://example.com/collections/foo/schema#/definitions/age"
+        },
+        "filter": {
+          "disallowed": true
```

Comment:
```suggestion
          "allowed": false
```
Negative property names are hard to understand because of the double negation

Reply:
I agree.  And am open to alternatives.  I think there is some value in having the default be the zero value though.  If we have an `allowed` property and it is absent, it would be nice if that were the same as `false`.  This means that the default would be to have no properties be filterable.

Again, I can drop this and go with the idea that if `allowed` is absent, the default is allowed (but that is also kind of weird).